### PR TITLE
AddTaskCentrality - convert to static function

### PR DESCRIPTION
**Pull request by adriansev**

### CODE REVIEW

**Comment by fprino** [comment]
Hi Adrian,

small comment: you do not need to increment the class revision number in the header file since you did not change the data members.

Cheers, Francesco

File: OADB/AliCentralitySelectionTask.h [hunk 1]
```diff
@@ -326,7 +327,7 @@ class AliCentralitySelectionTask : public AliAnalysisTaskSE {
   TH1F *fHOutVertex ;           //control histogram for vertex SPD
   TH1F *fHOutVertexT0 ;         //control histogram for vertex T0
 
-  ClassDef(AliCentralitySelectionTask, 31); 
+  ClassDef(AliCentralitySelectionTask, 32);
```
[the remark this answers is not in the document]
Comment (adriansev) [reply OADB/AliCentralitySelectionTask.h]: Hi, yes, of course, it was the habit :))

**Comment by adriansev** [comment]
@fprino so, should i fix the ClassDef to previous version? i was not clear to me if you need me to fix this..

**Comment by adriansev** [comment]
hmm, i have some trouble with syncing my branch with master ... do not test or merge please

**Comment by adriansev** [comment]
@fprino ok, done! could you check please? thank you!

**Comment by adriansev** [comment]
@fprino ping? the move to static AddTask would be useful later when using root6...

**Comment by dberzano** [comment]
@adriansev @fprino the idea of having a static method seems excellent to me, and I don't see any obvious drawback. This is a design pattern that leverages encapsulation (the steering macro can just be removed completely, but I think it needs to be kept for compatibility).

Again, I see no obvious problem here, if this approach works I think we should advertise it more broadly, maybe in our analysis tutorial (pinging @rbertens).

**Comment by raymondEhlers** [comment]
Just to add a few notes:

- We use this approach pretty heavily in the JE and EMCal groups and we haven't encountered issues. One just needs to be careful to update the AddTask macro arguments if you update the AddTask static method arguments (which shouldn't be a big issue here anyway)
- Even if the AddTask macro wasn't necessary for backwards compatibility with users loading the macro, the macro is still needed for the LEGO train

**Comment by adriansev** [comment]
@raymondEhlers : just not to take credit where is not mine :) : maybe i was the first to move the AddTask to cxx file but i use(d) a namespace in the form of NS_<my_task_name>. I think that @aiola had the idea with the usage of static class member.

**Comment by raymondEhlers** [comment]
@adriansev - I originally learned about it from Salvatore, and I remembered that he said you had some role it, but I didn't recall the details, so thanks for the correction!

**Pull request merged**